<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1378</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(223, 223, 223);
background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(24, 12, 7, 69), stop:0.208955 rgba(255, 0, 0, 69), stop:0.333333 rgba(255, 77, 55, 208), stop:0.472637 rgba(255, 111, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.477612 rgba(255, 178, 71, 130), stop:0.492537 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:0.711443 rgba(251, 255, 0, 145), stop:1 rgba(255, 255, 0, 69));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1351</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 7pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Add a Book</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>361</width>
        <height>511</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 6pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Add Book</string>
      </property>
      <widget class="QLineEdit" name="name_book">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Book Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="auther_book">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font:8pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Auther of the Book</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="number_page">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 7pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Number of Pages</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>210</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Categoreis</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cat">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
       </property>
       <item>
        <property name="text">
         <string>select category</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Travel and geography</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Business and economics</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Art and music</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Comics and graphic novels</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Computers and technology</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cooking and food</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Education and reference</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Entertainment</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Health and fitness</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hobbies and crafts</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Home and garden</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Language and literature</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Law and government</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Philosophy and religion</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Poetry and drama</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sports and recreation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Travel and geography</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="price">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>110</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>80</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>price</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="quant">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>170</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>170</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>quantity</string>
       </property>
      </widget>
      <widget class="QPushButton" name="eng">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>470</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>Engistrer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>470</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>release date:</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="rel_date">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>50</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>931</width>
        <height>511</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 7pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Books</string>
      </property>
      <widget class="QTableWidget" name="tab_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>881</width>
         <height>471</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);

background-color: rgb(255, 209, 129);
</string>
       </property>
       <column>
        <property name="text">
         <string>Book ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Book Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Number of pages</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>author</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Category</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>price</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>quantity</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QCheckBox" name="travel_4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>800</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 7pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Travel and geography</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1378</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCntrl_N">
   <property name="text">
    <string>Cntrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
